### PR TITLE
chore: Port more `AutomationPeers` from WinUI source

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): part of #14344 

## PR Type

What kind of change does this PR introduce?

- Feature
- Code style update (formatting)

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ToggleSwitch/ToggleSwitch.mux.cs [hunk 1]
```diff
@@ -358,22 +358,23 @@ private void AutomationToggleSwitchOnToggle()
...
-		//		static_cast<UIElement*>(this);
-
-		//	XPOINTF point;
-		//	static_cast<CUIElement*>(clickableElement.GetHandle()).GetClickablePointRasterizedClient(&point));
-
-		//	*result = { point.x, point.y };
-
-		//	return S_OK;
-		//}
+		public Point AutomationGetClickablePoint()
```
Comment (MartinZikmund) [inline src/Uno.UI/UI/Xaml/Controls/ToggleSwitch/ToggleSwitch.mux.cs:361]: This should be internal

File: src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs [hunk 2]
```diff
@@ -1443,6 +1455,66 @@ public void ScrollToHorizontalOffset(double offset)
...
+		/// Scroll content to the beginning.
+		/// </summary>
+		public void PageHome()
+			=> HandleVerticalScroll(ScrollEventType.First);
+
+		/// <summary>
+		/// Scroll content to the end.
+		/// </summary>
+		public void PageEnd()
+			=> HandleVerticalScroll(ScrollEventType.Last);
```
Comment (MartinZikmund) [inline src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs:1516]: These should be internal

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs (hunk 2 above)
Comment (morning4coffe-dev) [reply src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs]: Thanks @MartinZikmund!

**Pull request merged**